weasyl/controllers/user.py

```diff
@@ -109,9 +111,9 @@ def signup_post_(request):
     form = request.web_input(
         username="", password="", passcheck="", email="", emailcheck="",
         day="", month="", year="", recaptcha_challenge_field="",
-        g_recaptcha_response=request.web_input("g-recaptcha-response"))
+        g_recaptcha_response="")
```

Comment:
Yeah, that wasn’t the right way to fix that. The kwargs passed to `web.input()` are defaults for fields with the same name. If a field is required, you can just leave it off, or pass it as an unnamed argument.

```python
form = request.web_input('g-recaptcha-response', …)
```

and access as `form['g-recaptcha-response']`.